ssr/choo.js

```diff
@@ -63,17 +63,25 @@ module.exports.listRoutes = function (app) {
 module.exports.render = function (app, route, cb) {
   var state = {}
   state._experimental_prefetch = []
-  app.toString(route, state)
 
-  // TODO: replace with p-wait-all, once it knows how to handle an empty array.
-  Promise.all(state._experimental_prefetch)
-    .then(render, render)
+  return new Promise(function (resolve, reject) {
+    // First pass to populate prefetch
+    try {
+      app.toString(route, state)
+    } catch (err) {
+      // logging full stack trace to aid with debugging, sorry ui.
```

Comment:
is this acceptable? Was looking around to see if I could put the stacktrace in the browser, but got lost.